devtools/src/codegen/flexhtx/mod.rs

```diff
@@ -0,0 +1,121 @@
+use bimap::BiHashMap;
+use minidsp::formats::xml_config::Setting;
+use strong_xml::XmlRead;
+
+use super::spec::*;
+
+pub struct Target {}
+impl crate::Target for Target {
+    fn filename() -> &'static str {
+        "flexhtx.rs"
+    }
+
+    fn symbols() -> bimap::BiMap<String, usize> {
+        symbols()
+    }
+
+    fn device() -> Device {
+        device()
+    }
+}
+
+pub(crate) fn input(input: usize) -> Input {
+    Input {
+        gate: Some(Gate {
+            enable: format!("BM_DGain_{}_status", input + 1),
+            gain: Some(format!("BM_DGain_{}", input + 1)),
+        }),
+        meter: Some(format!("Meter_In_{}", input + 1)),
+        //meter_d: Some(format!("Meter_D_In_{}", input + 1)),
+        peq: vec![], //no input peq
+        /*bass_management: Some(Gate {
+            enable: format!("BM_DGain_{}_status", 3 + output),
+            gain: Some(format!("BM_DGain_{}", 3 + output)),
+        }),*/
+        routing: (0..7usize)
+            .map(|output| Gate {
+                enable: format!("BM_Mixer_{}_{}_status", input + 1, output + 1),
+                gain: Some(format!("BM_Mixer_{}_{}", input + 1, output + 1)),
+                //polarity: Some(format!("BM_Mixer_{input}_{output}_pol")),
+            })
+            .collect(),
+    }
+}
+
+pub(crate) fn output(output: usize) -> Output {
+    Output {
+        gate: Gate {
+            enable: format!("DGain_{}_0_status", output + 1),
+            gain: Some(format!("DGain_{}_0", output + 1)),
+        },
+        /*routing: (0..7usize) //input number
```

Comment:
some attempts to modify the output to conform with HTx additions (eg, the matrix routing)

Reply:
I assume these did not work? It looks like it's the right format.